src/azure-cli/azure/cli/command_modules/vm/_actions.py

```diff
@@ -40,15 +40,28 @@ def load_images_thru_services(cli_ctx, publisher, offer, sku, location):
         location = get_one_of_subscription_locations(cli_ctx)
 
     def _load_images_from_publisher(publisher):
-        offers = client.virtual_machine_images.list_offers(location, publisher)
+        from msrestazure.azure_exceptions import CloudError
+        try:
+            offers = client.virtual_machine_images.list_offers(location, publisher)
+        except CloudError as e:
+            logger.warning(str(e))
+            return
         if offer:
             offers = [o for o in offers if _matched(offer, o.name)]
         for o in offers:
-            skus = client.virtual_machine_images.list_skus(location, publisher, o.name)
+            try:
+                skus = client.virtual_machine_images.list_skus(location, publisher, o.name)
+            except CloudError as e:
+                logger.warning(str(e))
```

Comment:
if meet exception for some `offer`, the skus of it will be skipped ?

Reply:
Yes.